gradle/libs.versions.toml

```diff
@@ -204,7 +203,6 @@ moneta = { module = "org.javamoney:moneta", version.ref = "moneta" }
 
 okhttp-loggingInterceptor = { module = "com.squareup.okhttp3:logging-interceptor", version.ref = "okhttp" }
 okhttp-mockWebServer = { module = "com.squareup.okhttp3:mockwebserver", version.ref = "okhttp" }
-okhttp-coroutines = { module = "ru.gildor.coroutines:kotlin-coroutines-okhttp", version.ref = "okhttpCoroutines" }
```

Comment:
Why did we stop using this? 🤔

Reply:
It was messing up the IDE hints since this was pulling in an older version of OkHttp and then we were getting wrong errors when we were accessing the body of a okhttp Response. After removing this it now refers to the correct functions without wrong IDE hints.
The fact that everything was still working was just because the function names were changed in a backwards compatible way, but the type hints we were getting all this time were actually wrong. Particularly, we were getting `.body` as a `String` but it is actually a `ResponseBody` which just now I realized that then when we do `.string()` to it, it under the hood uses it properly, like doing `use {}` on it since it's `Closeable` and all that stuff.
The library is a 4-5 year old one anyway which isn't getting updated anyway, and it only provided one function, so I simply pulled that in to our code-base instead, I thought the tradeoff seemed logical to improve our IDE experience

Reply:
Makes sense!